mlflow/store/tracking/file_store.py

```diff
@@ -615,6 +617,9 @@ def create_run(self, experiment_id, user_id, start_time, tags, run_name):
         mkdir(run_dir, FileStore.METRICS_FOLDER_NAME)
         mkdir(run_dir, FileStore.PARAMS_FOLDER_NAME)
         mkdir(run_dir, FileStore.ARTIFACTS_FOLDER_NAME)
+        tags = tags or []
+        if run_name is not None:
```

Comment:
Is it ever not going to be `None` any longer?

Reply:
On line 596, we run `run_name = run_name if run_name is not None else _generate_random_name()`. `run_name` never becomes None.

Reply:
```suggestion
```